frontend/public/components/utils/cog.tsx

```diff
@@ -107,12 +107,12 @@ export class Cog extends DropdownMixin {
       { isDisabled ?
         <Tooltip content="disabled">
           <div ref={this.dropdownElement} className={classNames('co-m-cog', {'co-m-cog--disabled' : isDisabled})} >
-            <span role="button" tabIndex={0} className={classNames('fa', 'fa-ellipsis-v', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
+            <span className={classNames('fa', 'fa-ellipsis-v', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
             <span className="sr-only">Actions</span>
           </div>
         </Tooltip> :
         <div ref={this.dropdownElement} onClick={this.toggle} className={classNames('co-m-cog', {'co-m-cog--disabled' : isDisabled})} >
-          <span role="button" tabIndex={0} className={classNames('fa', 'fa-ellipsis-v', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
+          <button aria-haspopup="true" className="btn co-m-cog__button"><span className={classNames('fa', 'fa-ellipsis-v', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})}></span></button>
```

Comment:
Add `type="button" aria-label="Actions"`

If we make this change, the click handler should b on the button, not the div